utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -653,60 +665,154 @@ int main(int argc, char **argv)
+                                    colors32f[idx + j] = (Rpp32f)(colorBuffer[idx + j] * ONE_OVER_255);
+                                else if (inputBitDepth == 5)
+                                    colors8s[idx + j] = (Rpp8s)(colorBuffer[idx + j] - 128);
+                            }
+                        }
+                        else
+                        {
+                            int idx = boxesInEachImage * i;
+                            colorBuffer[idx] = 240;
+                            colorBuffer[idx + 1] = 120;
+                            colorBuffer[idx + 2] = 60;
+
+                            for (int j = 0; j < 3; j++)
+                            {
+                                if (!inputBitDepth)
+                                    colors8u[idx + j] = (Rpp8u)(colorBuffer[idx + j]);
+                                else if (inputBitDepth == 1)
+                                    colors16f[idx + j] = (Rpp16f)(
+                                        colorBuffer[idx + j] * ONE_OVER_255);
+                                else if (inputBitDepth == 2)
+                                    colors32f[idx + j] = (Rpp32f)(colorBuffer[idx + j] * ONE_OVER_255);
+                                else if (inputBitDepth == 5)
+                                    colors8s[idx + j] = (Rpp8s)(colorBuffer[idx + j] - 128);
+                            }
+                        }
                     }
 
                     startWallTime = omp_get_wtime();
-                    if (inputBitDepth == 0 || inputBitDepth == 1 || inputBitDepth == 2 || inputBitDepth == 5)
-                        rppt_crop_and_patch_gpu(d_input, d_input_second, srcDescPtr, d_output, dstDescPtr, roiTensorPtrSrc, cropRoi, patchRoi, roiTypeSrc, handle);
+                    if (!inputBitDepth)
+                        rppt_erase_gpu(d_input, srcDescPtr, d_output, dstDescPtr, anchorBoxInfoTensor, colors8u, numOfBoxes, roiTensorPtrSrc, roiTypeSrc, handle);
+                    else if (inputBitDepth == 1)
+                        rppt_erase_gpu(d_input, srcDescPtr, d_output, dstDescPtr, anchorBoxInfoTensor, colors16f, numOfBoxes, roiTensorPtrSrc, roiTypeSrc, handle);
+                    else if (inputBitDepth == 2)
+                        rppt_erase_gpu(d_input, srcDescPtr, d_output, dstDescPtr, anchorBoxInfoTensor, colors32f, numOfBoxes, roiTensorPtrSrc, roiTypeSrc, handle);
+                    else if (inputBitDepth == 5)
+                        rppt_erase_gpu(d_input, srcDescPtr, d_output, dstDescPtr, anchorBoxInfoTensor, colors8s, numOfBoxes, roiTensorPtrSrc, roiTypeSrc, handle);
                     else
                         missingFuncFlag = 1;
 
-                    break;
-                }
-                case 34:
+                break;
+            }
+            case 33:
+            {
+                testCaseName = "crop_and_patch";
+                for (i = 0; i < batchSize; i++)
```

Comment:
Why are all these changes are showing up @snehaa8

Reply:
Please fix tabbing for case 33 and 34.

Reply:
I pulled in latest develop and resolved conflicts directly in PR looks like its creating issues. I will fix these.